vita3k/modules/SceNpManager/SceNpManager.cpp

```diff
@@ -64,9 +64,11 @@ EXPORT(int, sceNpCheckCallback) {
     return STUBBED("Stub");
 }
 
-EXPORT(int, sceNpGetServiceState) {
+EXPORT(int, sceNpGetServiceState, SceNpServiceState *state) {
     TRACY_FUNC(sceNpGetServiceState);
```

Comment:
Not really important, but add parameter in tracy log

Reply:
Ah yes lol